ni/test/test_github.py

```diff
@@ -78,7 +79,16 @@ def test_ping(self):
                                                    util.FakeRequest(payload)))
         self.assertEqual(cm.exception.response.status, 200)
 
-    def test_process_skipping(self):
+    def test_verify_signature(self):
+        data = b'the-knights-who-say-ni'
+        secret_token = 'some_secret_token'
+        expected = '0b63e1a89ba1a9d18b4b19749431e244c93c7873'
```

Comment:
do you mean the function call I used to generate the expected hash? 
# mac = hmac.new(secret_token.encode(encoding='UTF-8'), msg=data, digestmod=hashlib.sha1).hexdigest()